util/regexp-assemble/data/942280.data

```diff
@@ -3,10 +3,18 @@
 
 ##!+ i
 
+##!> template semicolon ;
+
+##! The following part is going to be appended
+##! to match the end of a query or a comment syntax
+##!> template sql_comments (?:[#{{semicolon}}{]|\/\*|--)
+
+##! In PostgreSQL, you can use the pg_sleep() function
+##! to delay execution for a given number of seconds
 select\s*?pg_sleep
```

Comment:
Maybe you want to add a template like `optional-empty-space \s*?` ? If not, LGTM.

Reply:
that's a good idea, going to add it